src/components/DisplayComponent.js

```diff
@@ -17,6 +18,10 @@ class DisplayComponent extends Component {
       currentPopulation: 0,
       currentLevel: null,
       data: {},
+      curPage: 1,
+      totalPageNumber: 10, //0,
```

Comment:
Why is total page hardcoded to 10?

Reply:
it's a placeholder. It gets overwritten as soon as the code runs.

Reply:
If it's the case then I think we should make it 0 to avoid confusion.

Reply:
ok.